aries/aries_vcx_core/src/errors/mapping_anoncreds.rs

```diff
@@ -0,0 +1,41 @@
+use anoncreds::{Error as AnoncredsError, ErrorKind};
+
+use crate::errors::error::{AriesVcxCoreError, AriesVcxCoreErrorKind};
+
+impl From<AnoncredsError> for AriesVcxCoreError {
+    fn from(err: AnoncredsError) -> Self {
+        let cause = if err.kind() == ErrorKind::Input {
+            err.cause
+                .as_ref()
+                .and_then(|x| x.downcast_ref::<AnoncredsError>())
+        } else {
+            None
+        };
```

Comment:
Why does Input error from `anoncreds-rs` get special handling?

Reply:
Good catch - I don't know to be honest :) Removed